src/animators/animator.ts

```diff
@@ -1,5 +1,39 @@
 module Plottable {
+export module Animators {
```

Comment:
This one doesn't quite correspond to the pattern, it's like Drawer.ts with some elements in Animators and one in Plottable.
I have done all the new classes this way though.